crates/figma_import/src/bin/dcf_info.rs

```diff
@@ -12,11 +12,17 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-// Utility program to parse a .dcf file and print out the header and version info for the file.
+// Utility program to parse a .dcf file and print its contents.
+// By default prints the header and file info.
+// Provide the optional `--node` or `-n` switch to dump figma document using
+// that node as root.
+// Example:
+// `cargo run --bin dcf_info --features="dcf_info" -- tests/layout-unit-tests.dcf`
+// or
+// `cargo run --bin dcf_info --features="dcf_info" -- tests/layout-unit-tests.dcf -n HorizontalFill``
```

Comment:
Think you've got an extra ` mark at the end

Reply:
fixed.